src/hooks/index.js

```diff
@@ -744,54 +895,77 @@ exports.sendEmail = (context, maildata) => {
 
 				_.uniq(receipients).forEach((email) => {
 					if (!maildata.content.text && !maildata.content.html) {
-						logger.warning("(1) No mailcontent (text/html) was given. Don't send a mail.");
+						logger.warning(
+							"(1) No mailcontent (text/html) was given. Don't send a mail.",
+						);
 					} else {
-						mailService.create({
-							email,
-							replyEmail,
-							subject: maildata.subject || 'E-Mail von der Schul-Cloud',
-							headers: maildata.headers || {},
-							content: {
-								text: maildata.content.text
-									|| 'No alternative mailtext provided. Expected: HTML Template Mail.',
-								html: '', // still todo, html template mails
-							},
-							attachments,
-						}).catch((err) => {
-							logger.warning(err);
-							throw new BadRequest(
-								(err.error || {}).message
-								|| err.message
-								|| err
-								|| 'Unknown mailing error',
-							);
-						});
+						mailService
+							.create({
+								email,
+								replyEmail,
+								subject:
+									maildata.subject ||
+									'E-Mail von der Schul-Cloud',
+								headers: maildata.headers || {},
+								content: {
+									text:
+										maildata.content.text ||
+										'No alternative mailtext provided. Expected: HTML Template Mail.',
+									html: '', // still todo, html template mails
+								},
+								attachments,
+							})
+							.catch((err) => {
+								logger.warning(err);
+								throw new BadRequest(
+									(err.error || {}).message ||
+										err.message ||
+										err ||
```

Comment:
Codacy found an issue: ['||' should be placed at the beginning of the line.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045787)